Testing/Functional/Products/Tests/CallProductTests.ps1

```diff
@@ -0,0 +1,30 @@
+# The purpose of this script is to enable a GitHub Action workflow to
+# run the functional tests for all the products.
+# NOTE: At the moment it only does this for one product, Sharepoint.  Others
+#       will be added.
+# To run this script, call it from the root of the repo, like so:
+# ./Testing/Functional/Products/Tests/CallProductTests.ps1 <value of thumbprint>
+
+# The thumbprint of the cert used to access the product.
+$thumbprint = $args[0]
```

Comment:
Declare as a parameter to the script.

Reply:
I declared both parameters in a `param`.  Done.